react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -32,7 +38,13 @@ private void emitToJS(String eventName, @Nullable WritableMap data) {
     }
 
     public void getHtmlFromJS() {
-        emitToJS("requestGetHtml", null);
+        emitToJS(EVENT_NAME_REQUEST_GET_HTML, null);
+    }
+
+    public void updateHtml(String html) {
```

Comment:
I feel the `updateHtml` method name can be a little confusing in the context of the bridge since, it's not clear which way the update is going to happen. I think in this case the update is from the parent app to the RN side so, how about naming this "setHtmlInJS()` or similarly to make it more explicit?

Reply:
Yeah, let's do it. Fixed.

Reply:
> Fixed.

When a request / suggestion is addressed, it's far easier to read when the comment is answered to with the commit hash. Could you then also put a reference to the commit where this is fixed here (valid for future changes as well ;) )?

Reply:
agh - hit `send` too quickly: thanks in advance! 👍